js/views/itemEditVw.js

```diff
@@ -160,23 +160,26 @@ module.exports = Backbone.View.extend({
       contentType: false,
       processData: false,
       data: formData,
-      success: function(data) {
+      success: function(data) { //TODO: Have JQuery parse the JSON directly in ajax call
+        var errorModal, hashArray, imageArray;
         data = JSON.parse(data);
+
         if (data.success === true){
-          var imageArray = __.clone(self.model.get("combinedImagesArray"));
-          var hashArray = __.clone(self.model.get("imageHashesToUpload"));
+          imageArray = __.clone(self.model.get("combinedImagesArray"));
```

Comment:
Why did you remove var here? This will hoist imageArray to the global scope, or any parent scope with an imageArray variable. we want to keep it in the success function.